smpp/pdu/pdu_types.py

```diff
@@ -126,7 +126,18 @@ def __repr__(self):
 LanguageIndicator = Enum('LanguageIndicator', list(constants.language_indicator_name_map.keys()))
 DisplayTime = Enum('DisplayTime', list(constants.display_time_name_map.keys()))
 MsAvailabilityStatus = Enum('MsAvailabilityStatus', list(constants.ms_availability_status_name_map.keys()))
-NetworkErrorCode = Enum('NetworkErrorCode', list(constants.network_error_code_name_map.keys()))
+
+
+NetworkErrorCodeNetworkType = Enum('NetworkErrorCodeNetworkType', list(constants.network_error_code_name_map.keys()))
+
+NetworkErrorCodeBase = namedtuple('NetworkErrorCode', 'networkType, value')
```

Comment:
```suggestion
NetworkErrorCodeBase = namedtuple('NetworkErrorCode', 'networkType', 'value')
```

Reply:
that is the wrong syntax for named tuples: https://docs.python.org/3/library/collections.html#collections.namedtuple

Reply:
Oh interesting I thought it was a typo.